torch/csrc/jit/ir/ir.cpp

```diff
@@ -128,12 +128,6 @@ static std::ostream& operator<<(
   return printValueRefs(out, nodes);
 }
 
-static std::ostream& operator<<(
```

Comment:
Wait, this could be used by a downstream extension that want to print out these values.

Reply:
Since it is a static function in a cpp source, extension shouldn't know it.